OrbitCore/SamplingProfiler.cpp

```diff
@@ -344,48 +354,15 @@ void SamplingProfiler::FillThreadSampleDataSampleReports() {
             100.f * it->second / threadSampleData.m_NumSamples;
       }
       function.m_Address = address;
-
-      std::shared_ptr<Module> module = m_Process->GetModuleFromAddress(address);
-      function.m_Module = module ? module->m_Name : "???";
+      function.m_Module = Capture::GAddressToModuleName[address];
```

Comment:
I'm not sure if that works since function and module address are not the same. `GetModuleFromAddress` has a different behaviour than `GAddressToModuleName[]`. The former calls `upper_bound` while the latter only checks if an entry for the current address exists. Or did you completely revamp that and I'm wrong?

Reply:
Never mind. It's fine. ;-)